test/e2e/load.go

```diff
@@ -66,7 +66,7 @@ var _ = Describe("Load capacity", func() {
 		// Terminating a namespace (deleting the remaining objects from it - which
 		// generally means events) can affect the current run. Thus we wait for all
 		// terminating namespace to be finally deleted before starting this test.
-		err = deleteTestingNS(c)
+		err = checkTestingNSDeletedExcept(c, "")
```

Comment:
Can you not replace all of the code above this, and most of the code below it with framework.beforeEach() (i.e. port this test to Framework)?